WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -202,20 +201,40 @@ class AztecPostViewController: UIViewController, PostEditor {
 
 
     /// Publish Button
-    fileprivate(set) lazy var publishButton: UIBarButtonItem = {
-        let button = UIBarButtonItem(title: self.postEditorStateContext.publishButtonText, style: WPStyleGuide.barButtonStyleForDone(), target: self, action: #selector(publishButtonTapped(sender:)))
+    fileprivate(set) lazy var publishButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.addTarget(self, action: #selector(publishButtonTapped(sender:)), for: .touchUpInside)
+        button.setTitle(self.postEditorStateContext.publishButtonText, for: .normal)
+        button.sizeToFit()
         button.isEnabled = self.postEditorStateContext.isPublishButtonEnabled
+        button.setContentHuggingPriority(UILayoutPriorityRequired, for: .horizontal)
+        return button
+    }()
+
+    /// Publish Button
+    fileprivate(set) lazy var publishBarButtonItem: UIBarButtonItem = {
+        let button = UIBarButtonItem(customView: self.publishButton)
 
         return button
     }()
 
 
+    fileprivate lazy var moreButton: UIButton = {
+        let image = Gridicon.iconOfType(.ellipsis)
+        let moreItem = UIButton(type: .system)
+        moreItem.setImage(image, for: .normal)
+        var frame = CGRect(origin: .zero, size: image.size)
+        moreItem.frame = frame
+        moreItem.accessibilityLabel = NSLocalizedString("More", comment: "Action button to display more available options")
+        moreItem.addTarget(self, action: #selector(moreWasPressed), for: .touchUpInside)
+        moreItem.setContentHuggingPriority(UILayoutPriorityRequired, for: .horizontal)
+        return moreItem
+    }()
     /// NavigationBar's More Button
     ///
     fileprivate lazy var moreBarButtonItem: UIBarButtonItem = {
-        let image = Gridicon.iconOfType(.ellipsis)
-        let moreItem = UIBarButtonItem(image: image, style: .plain, target: self, action: #selector(moreWasPressed))
-        moreItem.accessibilityLabel = NSLocalizedString("More", comment: "Action button to display more available options")
+        let moreItem = UIBarButtonItem(customView: self.moreButton)
```

Comment:
Should we still set the `accessibilityLabel` on `moreItem` here? We could just pass it through from the button:

`moreItem.accessibilityLabel = self.moreButton.accessibilityLabel`

Reply:
Not sure about this one because there's a chance this may confuse users, if it's mentioned twice by VoiceOver.

As I understand it the item you interact with it really the button, so that's where I think the accessibility label should go.

Reply:
it's set on the button so it should be fine no?

Reply:
I just tested this and it looks like the accessibility label does get passed through 👍